storagegroup/storagegroup.go

```diff
@@ -252,7 +252,7 @@ func (sg *StorageGroup) UnmarshalJSON(data []byte) error {
 // error if object is not of TypeStorageGroup type.
 func ReadFromObject(sg *StorageGroup, o objectSDK.Object) error {
 	if typ := o.Type(); typ != objectSDK.TypeStorageGroup {
-		return fmt.Errorf("object is not of StorageGroup type: %s", typ)
+		return fmt.Errorf("object is not of StorageGroup type: %T", typ)
```

Comment:
we actually wanna see the value of `typ`. If fmt.Stringer is not provided lets use `%v` notation

Reply:
replaced with %v